3.7/arangosearch-analyzers.md

```diff
@@ -367,3 +364,50 @@ Name       | Type       | Language
 `text_ru`  | `text`     | Russian
 `text_sv`  | `text`     | Swedish
 `text_zh`  | `text`     | Chinese
+
+Supported Languages
+-------------------
+
+Analyzers rely on [ICU](http://site.icu-project.org/){:target="_blank"} for
+language-dependent tokenization and normalization. The ICU data file
+`icudtl.dat` that ArangoDB ships with contains information for a lot of
+languages, which are technically all supported.
+
+{% hint 'warning' %}
+The alphabetical order of characters is not taken into account by ArangoSearch,
+i.e. range queries and sorting of Views will not follow the language rules as
+per the defined Analyzer locale or the server startup option
+`--default-language`!
+{% endhint %}
```

Comment:
I think that'd be better to attach this notice to the description of supported SEARCH functions.
`IN_RANGE, >, >=, <, <=, STARTS_WITH` are affected